fairing/preprocessors/function.py

```diff
@@ -2,15 +2,23 @@
 import fairing
 import glob
 import os
+import types
 import tempfile
 from pathlib import Path
+from enum import Enum
 
 from fairing.constants import constants
 from .base import BasePreProcessor
+from fairing.functions.function_shim import get_execution_obj_type, ObjectType
 
 FUNCTION_SHIM = 'function_shim.py'
 SERIALIZED_FN_FILE = 'pickled_fn.p'
 
+# TODO(@karthikv2k): Ref #122 Find a better way to support deployer specific preprocessing
+OUPUT_FILE = """import cloudpickle
```

Comment:
Can you explain this?

Reply:
Seldon needs a module with the same name as the model class. For example it needs HouseServe class in a HouseServe.py file or in a HouseServe module. This type of requirement should be handled in a deployer specific preprocessing (#122 ) . For lack of it, I am adding it as part of function preprocessing. This is a no-op for training. Will remove it once #122 is fixed.